ReactCommon/cxxreact/ReactNativeVersion.h

```diff
@@ -15,9 +15,9 @@
 namespace facebook::react {
 
 constexpr struct {
-  int32_t Major = 1000;
-  int32_t Minor = 0;
-  int32_t Patch = 0;
+  int32_t Major = 0;
```

Comment:
we've never cared the core react native version. now we do?

Reply:
No. I think I can update all these to 0.71.0. I didn't think it _quite_ mattered where we start from but worth being consistent!